IlmBase/HalfTest/CMakeLists.txt

```diff
@@ -13,4 +13,9 @@ ADD_EXECUTABLE ( HalfTest
 
 TARGET_LINK_LIBRARIES ( HalfTest Half )
 
+INSTALL ( TARGETS
```

Comment:
Why install targets for the tests?  I thought cmake was able to run them on windows as part of the build process.